[2024-02-03T02:08:21.811+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-02T22:00:00+00:00 [queued]>
[2024-02-03T02:08:21.854+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-02T22:00:00+00:00 [queued]>
[2024-02-03T02:08:21.854+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-03T02:08:21.986+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-02-02 22:00:00+00:00
[2024-02-03T02:08:21.993+0000] {standard_task_runner.py:60} INFO - Started process 632 to run task
[2024-02-03T02:08:22.012+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-02-02T22:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp6gaauhx9']
[2024-02-03T02:08:22.041+0000] {standard_task_runner.py:88} INFO - Job 176: Subtask get_weather_data
[2024-02-03T02:08:22.187+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-02T22:00:00+00:00 [running]> on host d24a8cdd2842
[2024-02-03T02:08:22.468+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-02T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-02T22:00:00+00:00'
[2024-02-03T02:08:22.614+0000] {logging_mixin.py:188} INFO - https://api.openweathermap.org/data/2.5/weather?q=bangkok&appid=72832d0f2a3eed2b2cc39bf1c7c4f9d5&units=metric
[2024-02-03T02:08:22.614+0000] {logging_mixin.py:188} INFO - {'coord': {'lon': 100.5167, 'lat': 13.75}, 'weather': [{'id': 803, 'main': 'Clouds', 'description': 'broken clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 28.94, 'feels_like': 35.01, 'temp_min': 27.94, 'temp_max': 32.17, 'pressure': 1015, 'humidity': 82, 'sea_level': 1015, 'grnd_level': 1013}, 'visibility': 10000, 'wind': {'speed': 3.68, 'deg': 193, 'gust': 4.69}, 'clouds': {'all': 83}, 'dt': 1706925871, 'sys': {'type': 2, 'id': 2032756, 'country': 'TH', 'sunrise': 1706917497, 'sunset': 1706959097}, 'timezone': 25200, 'id': 1609350, 'name': 'Bangkok', 'cod': 200}
[2024-02-03T02:08:22.616+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-03T02:08:22.638+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240202T220000, start_date=20240203T020821, end_date=20240203T020822
[2024-02-03T02:08:22.714+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-03T02:08:22.737+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
